sdk/java-sdk-spring/src/main/scala/kalix/javasdk/impl/JwtDescriptorFactory.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.javasdk.impl
+
+import java.lang.reflect.Method
+import java.lang.reflect.Modifier
+
+import kalix.JwtMethodOptions
+import kalix.JwtServiceOptions
+import kalix.MethodOptions
+import kalix.javasdk.annotations.JWT
+import Reflect.Syntax._
+
+object JwtDescriptorFactory {
```

Comment:
Moved all the JWT annotation logic to its own class. To follow what we did for ACL.